awspub/image.py

```diff
@@ -414,7 +414,7 @@ def create(self) -> Dict[str, _ImageInfo]:
 
                 register_image_kwargs = dict(
                     Name=self.image_name,
-                    Description=self.conf.get("description", ""),
+                    Description=self.conf["description"],
```

Comment:
why are you changing this?

Reply:
But the type is `Optional[str]` which makes me think it can be omitted

Reply:
but the default is `None` then

Reply:
ok but before it was `""`. But I am fine with this if you are.

Reply:
let me double check what will happen if the description is not set...

Reply:
you are right:
```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter Description, value: None, type: <class 'NoneType'>, valid types: <class 'str'>
```

that's the error I get when not having a description set.

Reply:
I dropped that change now.